<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="0" skipped="1" tests="2" time="0.430" timestamp="2024-11-29T10:12:42.413416-06:00" hostname="US-5CD4021CL1-L"><testcase classname="test_api" name="test_skip" time="0.004"><skipped type="pytest.skip" message="no way of currently testing this">C:\Users\euiyoung.hwang\Git_Workspace\Vector_DB_with_LLM\tests\test_api.py:5: no way of currently testing this</skipped></testcase><testcase classname="test_api" name="test_api" time="0.107" /></testsuite></testsuites>